platform/darwin/src/MGLLoggingConfiguration.m

```diff
@@ -42,21 +42,66 @@ - (MGLLoggingBlockHandler)defaultBlockHandler {
         
         if (@available(iOS 10.0, macOS 10.12.0, *)) {
             static dispatch_once_t once;
-            static os_log_t mapbox_log;
+            static os_log_t info_log;
+            static os_log_t debug_log;
+            static os_log_t error_log;
+            static os_log_t fault_log;
             static os_log_type_t log_types[] = { OS_LOG_TYPE_DEFAULT,
                                                     OS_LOG_TYPE_INFO,
                                                     OS_LOG_TYPE_DEBUG,
                                                     OS_LOG_TYPE_ERROR,
                                                     OS_LOG_TYPE_FAULT };
             dispatch_once(&once, ^ {
-                mapbox_log = os_log_create("com.mapbox.maps-ios-sdk", "SDK");
+                info_log = os_log_create("com.mapbox.maps", "INFO");
```

Comment:
What is doing is create a category for the subsystem to prefix the messages to their respective logging level:
```
2018-11-13 10:48:56.423978-0800 Mapbox GL[57475:17412178] [INFO] -[MGLOfflineStorage reloadPacks] - 449: Reloading packs.
2018-11-13 10:48:56.427753-0800 Mapbox GL[57475:17412178] [DEBUG] -[MGLMapView setLocationManager:] - 4914: Setting locationManager: (null)
```
Otherwise is difficult to know what type of message is this.